templates/blazor-base/csharp/default/GettingStarted.txt

```diff
@@ -1,12 +1,25 @@
 Welcome to the Teams Toolkit! Here are some tips to get started building your app.
 
+Prerequisites
+----------
+
+If the project contains Bot capability, you will need to setup Ngrok(https://ngrok.com/) in order to tunnel from the Teams client to localhost.
+
+After install Ngrok please run the command in Command Prompt: ngrok http 2544
+
 
 Quick start
 ----------
 
-Select 'Debug > Start Debugging' or press F5. After press F5, in the browser, click Add to add your app to Teams.
+Before you run or Start Debugging for the first time, follow these steps:
 
-Learn more about developing Teams applications with Blazor at https://aka.ms/blazor-teams-app-dev.
+1. If the project contains Bot capability, check the Ngrok is running in your local environment. If the project only contains Tab capability, it doesn't need Ngrok.
+
+2. Select the Project > TeamsFx > Configure Microsoft Teams app. When prompted, sign in your M365 account. You can find the log in Output window.
+ 
+3. Select Debug > Start Debugging or press F5.
+
+4. After press F5, in the browser, click Add to add your app to Teams.
```

Comment:
You can add the sentence like:
If you want to know more about SDK usage, please check the GitHub page(https://github.com/OfficeDev/TeamsFx/tree/dev/packages/dotnet-sdk).